README.md

```diff
@@ -1,23 +1,115 @@
+# Getting started
 
+## Local install for development:
 
-A Flask based web service used by the public to get access to Civil Legal Aid services.
+### Setup
 
-# MOJ Flask Skeleton
+Create a virtual environment and install the python dependencies:
 
-The MOJ Flask Skeleton is set as a subtree from https://github.com/ministryofjustice/moj-flask-govuk-skeleton and is kept in the laa_access_civil_legal_aid folder.
```

Comment:
If we are using a subtree should we keep this documentation, along with 
```
To retrieve the most recent updates from the flask skeleton to your feature branch, run:
git subtree pull --prefix=laa_access_civil_legal_aid https://github.com/ministryofjustice/moj-flask-govuk-skeleton.git main --squash
``` 

It can belong in it's own section of the readme, however, if we want people to use it we should document it.